Tasks/ShellScript/task.json

```diff
@@ -49,6 +49,7 @@
         },
         {
             "name": "disableAutoCwd",
+            "aliases": ["disableAutoWorkingDirectory"],
```

Comment:
this is the old major verison (not deleted from master because reference docs still point to it), I would just revert this file.